### PR TITLE
Cargo economy balance

### PR DESCRIPTION
**Pull request by metalgearsloth**
First pass but will mean items don't sell for squat anymore. Branched from https://github.com/space-wizards/space-station-14/pull/11122

Resolves https://github.com/space-wizards/space-station-14/issues/8854

Hackmd for guidelines https://hackmd.io/@metalgearsloth/SJBS9UPei
Spreadsheet: https://docs.google.com/spreadsheets/d/1jJ94ilfr4m5e8JIDt4WQSE83fDXP-evDno5N8uzU8NU/edit#gid=0

Essentially everything is based around the base materials. Any categories I missed on my spreadsheet I just slightly bumped up the lathe price / cost price.

Botany / Chef still need proper balancing but better to wait for chef rework I think.

Some notes:
- Because of the abundance of tools most lockers are worth $200-$1,000 even though making tools isn't efficient but these aren't replenishable so I don't think it's too bad.

:cl:
- tweak: Sell prices have been significantly increased across the board so cargo should receive more income.
- tweak: Cargo passive income generation reduced from 10 per second to 2.
- tweak: Cargo shuttle cooldown reduced from 3 minutes to 30 seconds to facilitate their increased pace.

### CODE REVIEW

**Review by moonheart08 — APPROVED** [review]

**Comment by Partmedia** [comment]
I generally agree with the assumptions and goals in the design document. I think overall, these changes improve balance.

I'm afraid that when this is merged, cargo players are going to start looting everything of value on the station (vendors, closets, tools) to make money. Recently, cargo players have already been looting welding fuel and water tanks that could otherwise have been used for selling. **How do you plan on preventing wholesale looting of items that are going to be of value to the station?**

You may remember a change where gas canisters would be priced (#10129) to encourage cargo-atmos collaboration. Even though there are gas miners, players still have to put into the work of putting gas in cans and dragging them to cargo. Nonetheless, this had to be removed in spectacular fashion (#10948). **What's going to stop some of these prices from needing to be nerfed to the ground in not too much time?** (and have you _seen_ anyone selling synthesized gases since then?)

A few things to point out about specific changes:

- FoodBase is up to 50, but that means 20 pizzas only sells for 1000. How much does a cow cost?
- Why are you increasing the base price for clothing? Only HoP can make clothing. Are you suggesting that HoP participate in this cargo economy? I think what actually is going to happen is that cargo is going to loot uniform vendors.
- Can you justify why the cargo cooldown was changed from 150 seconds to 30 seconds?
- Why are human organ prices being increased?

**Comment by metalgearsloth** [comment]
Good questions

> Recently, cargo players have already been looting welding fuel and water tanks that could otherwise have been used for selling. How do you plan on preventing wholesale looting of items that are going to be of value to the station?

Cargo getting their hands on tanks is an intended mechanic from https://github.com/space-wizards/space-station-14/pull/9466

For most items cargo can just buy more of steel / glass given the crates are worth less than the crafted goods you can get out of them so replenishing stuff like tools shouldn't be a problemTM. I think the real thing that will dissuade them from looting everything not bolted down is the fact that everything is worth at least a token amount so there's not any specific items they will beeline.

> What's going to stop some of these prices from needing to be nerfed to the ground in not too much time? (and have you seen anyone selling synthesized gases since then?)

Most of the time I see cargo waiting to receive their passive income tbh. If any bad metas develop we can just adjust or revert as needed like usual.

> FoodBase is up to 50
Food isn't balanced because chef rework is going to throw it all out the window in the next few months (probably) anyway. Considering we want to emphasis crafted goods all of the recipes are gonna change to reflect effort put in.

> Why are you increasing the base price for clothing?
To at least give cloth some value but I have dropped it by half. If the hop has enough freetime to make clothes then I think that's useful. Worst case they loot the 1 or 2 clothing machines around.
Alternatively we can drop the cloth price and drop uniforms even more.

> Can you justify why the cargo cooldown was changed from 150 seconds to 30 seconds
"3 minutes to 30 seconds to facilitate their increased pace." I could see 60 maybe but considering cargo needs to sell much more stuff and their shuttle is small I don't see an issue with it when I playtested it with tstalker. From memory in ss13 it's 60 which could also be used.

> Why are human organ prices being increased?
Add some dilemma between using it for biomass vs selling it.

**Comment by moonheart08** [comment]
> I generally agree with the assumptions and goals in the design document. I think overall, these changes improve balance.
> 
> I'm afraid that when this is merged, cargo players are going to start looting everything of value on the station (vendors, closets, tools) to make money. Recently, cargo players have already been looting welding fuel and water tanks that could otherwise have been used for selling. **How do you plan on preventing wholesale looting of items that are going to be of value to the station?**
> 
> You may remember a change where gas canisters would be priced (#10129) to encourage cargo-atmos collaboration. Even though there are gas miners, players still have to put into the work of putting gas in cans and dragging them to cargo. Nonetheless, this had to be removed in spectacular fashion (#10948). **What's going to stop some of these prices from needing to be nerfed to the ground in not too much time?** (and have you _seen_ anyone selling synthesized gases since then?)
> 
> A few things to point out about specific changes:
> 
>     * FoodBase is up to 50, but that means 20 pizzas only sells for 1000. How much does a cow cost?
> 
>     * Why are you increasing the base price for clothing? Only HoP can make clothing. Are you suggesting that HoP participate in this cargo economy? I think what actually is going to happen is that cargo is going to loot uniform vendors.
> 
>     * Can you justify why the cargo cooldown was changed from 150 seconds to 30 seconds?
> 
>     * Why are human organ prices being increased?

Organ prices were bad originally as a crude bit of balancing to avoid free stuff from the then infinite cloning.

File: Content.Client/UserInterface/StatValuesEui.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+namespace Content.Client.UserInterface;
+
+public sealed class StatValuesEui : BaseEui
+{
+    private readonly StatsWindow _window;
+
+    public StatValuesEui()
+    {
+        _window = new StatsWindow();
+        _window.Title = "Melee stats";
```
Comment (mirrorcult) [inline Content.Client/UserInterface/StatValuesEui.cs:14]: loc

**Pull request merged**